src/apps/intel/intel10g.lua

```diff
@@ -452,22 +294,10 @@ end
 local txdesc_flags = bits{ifcs=25, dext=29, dtyp0=20, dtyp1=21, eop=24}
 
 function M_sf:transmit (p)
-   -- We must not send packets that are bigger than the MTU.  This
-   -- check is currently disabled to satisfy some selftests until
-   -- agreement on this strategy is reached.
-   -- if p.length > self.mtu then
-   --    if self.snmp then
-   --          local errors = self.snmp.ifTable:ptr('ifOutDiscards')
-   --          errors[0] = errors[0] + 1
-   --    end
-   --    packet.free(p)
-   -- else
-   do
-      self.txdesc[self.tdt].address = memory.virtual_to_physical(p.data)
-      self.txdesc[self.tdt].options = bor(p.length, txdesc_flags, lshift(p.length+0ULL, 46))
-      self.txpackets[self.tdt] = p
-      self.tdt = band(self.tdt + 1, num_descriptors - 1)
-   end
+   self.txdesc[self.tdt].address = memory.virtual_to_physical(p.data)
+   self.txdesc[self.tdt].options = bor(p.length, txdesc_flags, lshift(p.length+0ULL, 46))
```

Comment:
In future patches, I'd like comments or defined constants explaining numbers like that 46.